### ISSUE TITLE
Wrong box status, showing "running" instead of "Powered Off"

### ISSUE THREAD

**Issue by GabLeRoux** [vaibhav-sidapara/vagrant-manager#3]
Very nice repos btw, good job ;)
I tried it and I think the way it gets the box names is kind of wrong and for some reason, it seems to get the wrong box status from the vms. See the following screenshot;

![image](https://cloud.githubusercontent.com/assets/1264761/21653496/692d9828-d27f-11e6-8caa-6a9c7a840507.png)

In fact, I only have one vagrant vm running right now, but many of them are marked as status running (green icon). 

![image](https://cloud.githubusercontent.com/assets/1264761/21653592/df4ee1b0-d27f-11e6-8175-79b3ffdff042.png)

I'm on latest commit 2d82e9b0803a3294d11ae5e60aa83c66957c9516

I may have a look at the code later and see how this could be improved :)

**Comment by vaibhav-sidapara** [comment]
Hi @GabLeRoux,

Can you try to run this command `vagrant global-status --prune`.

`global-status` - State all active vagrant environments on the system.
`--prune` - Prunes invalid entries from the list.

### Explanation

Vagrant Manager tries to read the status of the boxes from an index file which is generated by the vagrant itself. The above command will help to correct the states of the vagrant boxes. You can read more about the command [here](https://www.vagrantup.com/docs/cli/global-status.html).

Let me know if the problem still persists.

**Comment by GabLeRoux** [comment]
Thanks for your answer, I recently moved to Fedora and got rid of a lot of these vms but I could still reproduce. Your answer seems quite right. I just tried again, saw a vm that was supposed to be in `poweroff` state. I still had a vm saying it was up in vagrant but down in virtualbox, `prune` did not fix this, it didn't refresh `state`. 

Anyway, the results from the dropdown menu matchs the `global-status` command so problem is on Vagrant side.

Quote from ` vagrant global-status` command output:

> The above shows information about all known Vagrant environments on this machine. This data is cached and may not be completely up-to-date.

:+1: :tada: Oh and I really like that About popup! :dark_sunglasses:
